adapters/rubicon/rubicon.go

```diff
@@ -489,7 +489,16 @@ func (a *RubiconAdapter) Call(ctx context.Context, req *pbs.PBSRequest, bidder *
 		rubiReq.Device = &deviceCopy
 
 		if thisImp.Video != nil {
-			videoExt := rubiconVideoExt{Skip: params.Video.Skip, SkipDelay: params.Video.SkipDelay, RP: rubiconVideoExtRP{SizeID: params.Video.VideoSizeID}}
+
+			videoSizeId := params.Video.VideoSizeID
+			if videoSizeId == 0 {
+				resolvedSizeId, err := resolveVideoSizeId(thisImp.Video.Placement, thisImp.Instl, thisImp.ID)
+				if err == nil {
+					videoSizeId = resolvedSizeId
+				}
```

Comment:
In line 496, if the error turns out to not be `nil`, do we want to `continue` as we do in the check in line 505 below?
```
491         if thisImp.Video != nil {
492
493             videoSizeId := params.Video.VideoSizeID
494             if videoSizeId == 0 {
495                 resolvedSizeId, err := resolveVideoSizeId(thisImp.Video.Placement, thisImp.Instl, thisImp.ID)
496                 if err == nil {
497                     videoSizeId = resolvedSizeId
498 -               }
    +               } else { // err != nil
    +                   continue
    +               } 
499             }
500
501             videoExt := rubiconVideoExt{Skip: params.Video.Skip, SkipDelay: params.Video.SkipDelay, RP: rubiconVideoExtRP{SizeID: videoSizeId}}
502             thisImp.Video.Ext, err = json.Marshal(&videoExt)
503         } else {
504             primarySizeID, altSizeIDs, err := parseRubiconSizes(unit.Sizes)
505             if err != nil {
506                 continue
507             }
adapters/rubicon/rubicon.go
```